soccer/gameplay/tactics/stopped/circle_near_ball.py

```diff
@@ -155,11 +139,27 @@ def execute_running(self):
         for b in self.all_subbehaviors():
             if b.robot is not None:
                 num_robots += 1
+        #if the number of robots has changed, recreate move behaviors to match new number of robots
+        if (self.num_robots != num_robots):
+            self.num_robots = num_robots
+            self.remove_all_subbehaviors()
+            i = 0
+            for pt in range(6):
+                self.add_subbehavior(skills.move.Move(),
+                                     name="robot" + str(i),
+                                     required=False,
+                                     priority=6 - i)
```

Comment:
This can be `6-pt`, since pt increases from 0->5. It would clean up the `i=0` and `i+1` bits, they are kinda 'unpythonic'. I'm not sure what that means though.